webapp/src/app.tsx

```diff
@@ -3070,7 +3071,7 @@ export class ProjectView
                     </div>
                 </div>
                 <div id="maineditor" className={(sandbox ? "sandbox" : "") + (inDebugMode ? "debugging" : "")} role="main" aria-hidden={inHome}>
-                    {showCollapseButton && <sui.Button id='togglesim' className={`computer only collapse-button large`} icon={`inverted chevron ${this.state.collapseEditorTools ? 'right' : 'left'}`} title={collapseTooltip} onClick={this.toggleSimulatorCollapse} />}
+                    {showCollapseButton && <sui.Button id='togglesim' className={`computer only collapse-button large`} icon={`inverted chevron ${(this.state.collapseEditorTools || isRTL ) && !(this.state.collapseEditorTools && isRTL) ? 'right' : 'left'}`} title={collapseTooltip} onClick={this.toggleSimulatorCollapse} />}
```

Comment:
Let's move this complicated ternary into a variable with a descriptive name. I'm finding it hard to parse